src/services/crons/currency.converter.cache.warmer.service.ts

```diff
@@ -0,0 +1,51 @@
+import { Lock } from '@multiversx/sdk-nestjs-common';
+import { Inject, Injectable } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+import { RedisPubSub } from 'graphql-redis-subscriptions';
+import { CurrencyConverterComputeService } from 'src/modules/currency-converter/services/currency.converter.compute.service';
+import { CurrencyConverterSetterService } from 'src/modules/currency-converter/services/currency.converter.setter.service';
+import { PUB_SUB } from '../redis.pubSub.module';
+import { CurrencyCategory } from 'src/modules/currency-converter/models/currency.rate.model';
+
+@Injectable()
+export class CurrencyConverterCacheWarmerService {
+    constructor(
+        private readonly currencyConverterCompute: CurrencyConverterComputeService,
+        private readonly currencyConverterSetter: CurrencyConverterSetterService,
+        @Inject(PUB_SUB) private pubSub: RedisPubSub,
+    ) {}
+
+    @Cron(CronExpression.EVERY_HOUR)
```

Comment:
The cron job should run faster than the expiring keys from cache including redis cache. The TTL on redis is set to 1 hour, so the cron job should happen faster